articles/api/management/v2/tokens.md

```diff
@@ -1,86 +1,278 @@
+  "expires_in": 86400,
+  "scope": "read:clients create:clients read:client_keys",
+  "token_type": "Bearer"
+}
+```
 
-## Uses
+From the above we can see that our `access_token` is a [bearer access token](https://tools.ietf.org/html/rfc6750), it will expire in 24 hours (86400 seconds), and it has been authorized to read and create clients.
 
-The Auth0 Management APIv2 access token is used to call the Auth0 Management APIv2.  This token is required to update the app_metadata portions of the user profile.
+### 2. Use the Token
 
-## API Secret
+To use this token, just include it in the `Authorization` header of your request .
 
-Keep your API secret secure. In the event that your API secret has been compromised or you need to invalidate all of your tokens you can change the API secret.
+```har
+{
+  "method": "POST",
+  "url": "http://PATH_TO_THE_ENDPOINT/",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" },
+    { "name": "Authorization", "value": "Bearer YOUR_ACCESS_TOKEN"}
+  ]
+}
+```
 
-You can change the API secret  in the dashboard by visiting this URL directly:
+For example, in order to [Get all clients](/api/management/v2#!/Clients/get_clients) use the following:
+
+```har
+{
+  "method": "GET",
+  "url": "https://${account.namespace}/api/v2/clients",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" },
+    { "name": "Authorization", "value": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6Ik5ESTFNa05DTVRGQlJrVTRORVF6UXpFMk1qZEVNVVEzT1VORk5ESTVSVU5GUXpnM1FrRTFNdyJ9.eyJpc3MiOiJodHRwczovL2RlbW8tYWNjb3VudC5hdXRoMC5jb20vIiwic3ViIjoib9O7eVBnMmd4VGdMNjkxTnNXY2RUOEJ1SmMwS2NZSEVAY2xpZW50cyIsImF1ZCI6Imh0dHBzOi8vZGVtby1hY2NvdW50LmF1dGgwLmNvbS9hcGkvdjIvIiwiZXhwIjoxNDg3MDg2Mjg5LCJpYXQiOjE5ODY5OTk4ODksInNjb3BlIjoicmVhZDpjbGllbnRzIGNyZWF0ZTpjbGllbnRzIHJlYWQ6Y2xpZW50X2tleXMifQ.oKTT_cEA_U6hVzNYPCl_4-SnEXXvFSOMJbZyFydQDPml2KqBxVw_UPAXhjgtW8Kifc_b2HQ4jFh7nH0KC_j1XjfEJPvwFZgqfI_ILzO3DPfpEIK_n_aX-Tz4okbZe6nj2aT_qLpHimLxK50jOGaMuzp4a1djHJTj5q-NbIiPW8AJowS2-gveP4T3dyyegUsZkmTNwrreqppPApmpWWE-wVsxnVsI_FZFrHnq0rn7lmY_Iz6vyiZjaKrd2C3hFm0zFGTn8FslBfHUldTcDNzOKOpCq7HFMeU0urXBXDetrzkW1afxIqED3G2C51JEV-4nTRYUinnWgXJfLJ87G3ge_A"}
+  ]
+}
+```
 
-```text
-${manage_url}/#/applications/YOUR_API_ID/settings
+<div class="alert alert-info">
+  You can get the curl command for each endpoint from the Management API v2 Explorer. Go to the endpoint you want to call, and click the <em>get curl command</em> link at the <em>Test this endpoint</em> section.
+</div>
+
+That's it! You are done!
+
+### Sample Implementation: Python
+
+This python script gets a Management API v2 access token, uses it to call the [Get all clients](/api/management/v2#!/Clients/get_clients) endpoint, and prints the response in the console:
+
+```python
+def main():
+  import json, urllib, urllib2
+
+  # Configuration Values
+  DOMAIN = "${account.namespace}"
+  AUDIENCE = "https://${account.namespace}/api/v2/"
+  CLIENT_ID = "${account.clientId}"
```

Comment:
Maybe add a comment that explains that it's the client id and secret of the non interactive client created previously? and a note that it's the non interactive client the one who has to be authorized to access all clients by setting up the appropriate scope